node/src/BaseClient.ts

```diff
@@ -1056,6 +1057,27 @@ export class BaseClient {
         return this.createWritePromise(createLLen(key));
     }
 
+    /** Sets the list element at index to element.
+     * The index is zero-based, so 0 means the first element,1 the second element and so on.
+     * Negative indices can be used to designate elements starting at the tail of
+     * the list. Here, -1 means the last element, -2 means the penultimate and so forth.
+     * See https://valkey.io/commands/lset/ for details.
```

Comment:
```suggestion
     *
     * See https://valkey.io/commands/lset/ for details.
```

Reply:
same

Reply:
Personally, I like to add the extra line for easier to read. But since other Node commands are doing without the extra line space. You can just keep it as is.

Reply:
Yeah same as above, it would be good to align on this, the existing docs have it both ways but I prefer to have the extra space